test/Schema.js

```diff
@@ -154,9 +154,9 @@ describe('Schema tests', function (){
     schema.attributes.aObject.type.name.should.eql('object');
     should.exist(schema.attributes.aObject.default);
 
```

Comment:
Instead of changing these tests, it may be more useful to set useDocumentTypes and useNativeBooleans to false.